person.rb

```diff
@@ -0,0 +1,23 @@
+require 'securerandom'
+
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @id = SecureRandom.uuid
+    @parent_permission = parent_permission
+  end
+
+  def can_use_service?
+    of_age || parent_permission
+  end
+
+  private
+
+  def of_age
+    age >= 18
+  end
```

Comment:
- [ ] In Ruby, a question mark (?) at the end of a method name is a naming convention that's often used to indicate that the method returns a boolean value. These methods are commonly referred to as "predicate methods." Predicate methods typically return true or false based on some condition. Kindly add **`?`** to the method name. ℹ️ 

`def of_age?`